src/wasm/wasm.cpp

```diff
@@ -796,4 +796,13 @@ void Module::updateMaps() {
   }
 }
 
+uint64_t Table::Segment::constOffset() {
+  if (offset->is<GetGlobal>())
```

Comment:
this seems like surprising behavior given the name of the name of the function. are non-constant offsets not handled, then?

Reply:
(if so, that seems to make this very wasm2asm specific and not a general utility)